tests/test_merge.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+
+def assert_all_files_present(filenames):
+    assert all(os.path.exists(f) for f in filenames)
+
+
+def test_horizontal_merge(vcf_merger, unique_samples_vcfs):
+    vcfs = {'alias': unique_samples_vcfs}
+    filenames = vcf_merger.horizontal_merge(vcfs)
+    assert filenames == [os.path.join(vcf_merger.output_dir, 'alias_merged.vcf.gz')]
+    assert_all_files_present(filenames)
+
+
+def test_horizontal_merge_multiple_groups(vcf_merger, unique_samples_vcfs, unique_samples_vcfs_2):
+    vcfs = {'1': unique_samples_vcfs, '2': unique_samples_vcfs_2}
+    filenames = vcf_merger.horizontal_merge(vcfs)
+    assert sorted(filenames) == [
+        os.path.join(vcf_merger.output_dir, '1_merged.vcf.gz'),
+        os.path.join(vcf_merger.output_dir, '2_merged.vcf.gz')
+    ]
+    assert_all_files_present(filenames)
```

Comment:
This is a broader question for further discussion: should we have tests to ensure that the final merged file has the INFO fields and other data points (genotypes etc.,) correctly populated? A point that was brought up in a recent stand-up was that "bcftools merge" (or concat for that matter) could sometimes compromise the integrity of data in the source files. Also, Thomas brought up a point about dealing with multi-allelics. How do we address such issues?